libcst/_typed_visitor_base.py

```diff
@@ -10,12 +10,11 @@
 if TYPE_CHECKING:
     from libcst._typed_visitor import CSTTypedBaseFunctions  # noqa: F401
 
-T = TypeVar("T")
+FuncType = Callable
```

Comment:
Hmm, yeah. I know pyre is moving away from generics without a constraint, so this probably won't work going forward with pyre anyway.

Reply:
Does mypy require the extra level of indirection? If not, can you change the below line to just `F = TypeVar("F", bound=Callable)` directly and make this work on both?

Reply:
Ha didn’t mean to resolve that. This is a good idea, I’ll try it tomorrow.

Reply:
added in 05c86d8, thanks @DragonMinded :)